### DISCUSSION TITLE
Fake-IP 模式下，Play商店无法下载、更新App

### DISCUSSION THREAD

**Discussion by Rangtian** [vernesong/OpenClash#4064] [Q&A] [answered]
Fake-IP 混合模式，能正常使用 Youtube、Copilot 与 ChatGPT，也能正常访问Play商店，就是下载、更新App一直显示“等待中...”。
切换到 Redir-Host 模式，或使用手机里的Clash，都可以正常下载、更新App。
Dashboard里查看手机的链接信息，都是Redir，并没有Direct。
我该往哪个方向去排查问题？
坐等大佬，万分感谢！
![1](https://github.com/user-attachments/assets/9d75ceee-7eae-44bb-a078-729a2d01d4bf)
![2](https://github.com/user-attachments/assets/6e0837d5-2117-4a2d-a037-090c0552a64a)

**Comment by fouvromenth** [comment]
你往下翻就能看见有人讨论
https://github.com/vernesong/OpenClash/discussions/3131

**Reply by Rangtian** [reply]
搞定了，谢谢你！

**Comment by Rangtian** [comment, marked as answer]
我按照 #3131 中所说，把 googleapis.cn 添加到 插件设置 -> 流量控制 -> 绕过指定区域 IPv4 黑名单，就能正常下载、更新App了

![1](https://github.com/user-attachments/assets/cf3ae59e-adaf-4b1a-b9de-96b0f6e8f90c)
